README.md

```diff
@@ -10,8 +10,16 @@ npm i fastify-slow-down
 
 ## Usage
 
-Register the plugin.
-This plugin will add an `onRequest` hook to slow down replies if a client (based on their IP address by default) has made too many multiple requests in the given `timeWindow`.
+Register SlowDown as a Fastify plugin.
+This plugin will add an `onRequest` hook to slow down replies if a client (based on their IP address by default) has made too many multiple requests in the given `timeWindow` and
+it will add `slowDown` decoration to the `request` with the following information:
+
+| Name | Type | Description |
+|------|------|-------------|
+| limit | number | the maximum limit until the server starts to delay the response |
+| index request | number | the index of the current request|
+| remaining requests | number | how many requests are left until the server starts to delay the response |
+| delay | number, undefined | value of the delay (in milliseconds) applied to this request |
```

Comment:
1. use the property names, not bogus names!
2. use typescript syntax to describe the types, not an invented syntax